PS3GUD.py

```diff
@@ -256,9 +269,9 @@ def downloadFiles(self, window):
             sha1 = dl["sha1"]
             size = dl["size"]
             id = dl["gameid"]
-            fdir = os.path.join(self.getConfig("dldir")+"/", utils.filterIllegalCharsFilename(self.getTitleNameFromId(id))+"["+id+"]/")
```

Comment:
since some linux tools have problems with spaces in filenames/paths its preferable to use an underscore